src/main/java/com/mraof/minestuck/item/AncientThumbDrive.java

```diff
@@ -0,0 +1,46 @@
+package com.mraof.minestuck.item;
+
+import net.minecraft.MethodsReturnNonnullByDefault;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.network.chat.Component;
+import net.minecraft.sounds.SoundEvents;
+import net.minecraft.sounds.SoundSource;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResultHolder;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.*;
+import net.minecraft.world.level.Level;
+import org.jetbrains.annotations.Nullable;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.util.List;
+
+@ParametersAreNonnullByDefault
+@MethodsReturnNonnullByDefault
+public class AncientThumbDrive extends Item
+{
+	public AncientThumbDrive(Properties properties)
+	{
+		super(properties);
+	}
+	
+	@Override
+	public InteractionResultHolder<ItemStack> use(Level level, Player player, InteractionHand usedHand)
+	{
+		level.playSound(player, player.getX(), player.getY(), player.getZ(), SoundEvents.GRAVEL_STEP, SoundSource.PLAYERS, 0.01F, 0.3F);
+		level.playSound(player, player.getX(), player.getY(), player.getZ(), SoundEvents.ARMOR_EQUIP_IRON, SoundSource.PLAYERS, 0.02F, 0.6F);
```

Comment:
Volume is too low for these sound effects

Reply:
upped to 0.1, and 0.2 respectively. meant to be on the quieter side

Reply:
Its rather unintuitive of Minecraft, but what those volumes best refer to is how far it takes before the sound drops off. If you are at the source, its going to essentially be the same volume unless Im mistaken

Reply:
I vaguely recall something like that for volumes over 1.0 being clamped to 1.0, but I'm not sure what you said is the case when the volume is in the range 0.0 to 1.0